runtime/doc/vim_diff.txt

```diff
@@ -294,17 +373,9 @@ These Nvim features were later integrated into Vim.
 - 'statusline' supports unlimited alignment sections
 
 ==============================================================================
-5. Changed features					 *nvim-features-changed*
-
-Nvim always builds with all features, in contrast to Vim which may have
-certain features removed/added at compile-time. |feature-compile|
-
-Some Vim features were changed in Nvim, and vice versa.
+Changed features					 *nvim-changed*
 
-If a Python interpreter is available on your `$PATH`, |:python| and |:python3|
-are always available and may be used simultaneously. See |provider-python|.
-
-|:redir| nested in |execute()| works.
+This section lists various low-level details about other behavior changes.
```

Comment:
This section (`:h nvim-changed`) should rarely be used. Most "changes" are features. 

This section remains for documenting various notes.